website/docs/r/netapp_volume.html.markdown

```diff
@@ -113,6 +114,8 @@ The following arguments are supported:
 
 * `storage_quota_in_gb` - (Required) The maximum Storage Quota allowed for a file system in Gigabytes.
 
+* `snapshot_directory_visible` - (Optional) Specifies whether the snapshot path of a volume is visible.
```

Comment:
>snapshot [](http://example.com/codeflow?start=66&length=8)

proper value to show here is:

`.snapshot` (NFS clients) or `~snapshot` (SMB clients)